aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogCommitFailure.java

```diff
@@ -446,9 +426,8 @@ public void testOtherGlueException() {
 
     GlueTableOperations spyOps = Mockito.spy(ops);
     failCommitAndThrowException(spyOps, GlueException.builder().statusCode(300).build());
-
-    AssertHelpers.assertThrows(
-        null, GlueException.class, () -> spyOps.commit(metadataV2, metadataV1));
+    Assertions.assertThatThrownBy(() -> spyOps.commit(metadataV2, metadataV1))
+        .isInstanceOf(GlueException.class);
```

Comment:
and here

Reply:
I think here it's not needed as the exception without a message is thrown from the above line